README.md

```diff
@@ -16,23 +16,14 @@ Please make sure to run [eslint](http://eslint.org/) against any proposed change
 
 ## Usage
 
-To parse a document, you must create a new `Readability` object from a URI object and a document object, and then call `parse()`. Here's an example:
+To parse a document, you must create a new `Readability` object from a document object, and then call `parse()`. Here's an example:
 
 ```javascript
-var loc = document.location;
-var uri = {
-  spec: loc.href,
-  host: loc.host,
-  prePath: loc.protocol + "//" + loc.host,
-  scheme: loc.protocol.substr(0, loc.protocol.indexOf(":")),
-  pathBase: loc.protocol + "//" + loc.host + loc.pathname.substr(0, loc.pathname.lastIndexOf("/") + 1)
-};
-var article = new Readability(uri, document).parse();
+var article = new Readability(document).parse();
 ```
 
 This `article` object will contain the following properties:
 
-* `uri`: original `uri` object that was passed to constructor
```

Comment:
Can you then also remove the `uri` property from the returned result, please? :-)

Reply:
Done